typed-racket-test/succeed/with-handlers-star.rkt

```diff
@@ -0,0 +1,5 @@
+#lang typed/racket/base
+(with-handlers* ([exn:fail? (λ (e) 2)])
+  1)
+(with-handlers* ([exn:fail? (λ (e) 2)])
```

Comment:
Somehow this is causing the integration test suite to timeout.

Reply:
It was caused by a typo : 
```racket
(define-syntax with-handlers*: (make-with-handlers #'with-handlers*:))
```